sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalProcessStreamConsumer.java

```diff
@@ -32,13 +32,15 @@
 import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.utils.log.Logger;
 import org.sonar.api.utils.log.Loggers;
+import org.sonarsource.api.sonarlint.SonarLintSide;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonarsource.api.sonarlint.SonarLintSide'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=misc&issues=AWEykIdRr8h_fVHZ4rNu&open=AWEykIdRr8h_fVHZ4rNu)

[ISSUE_KEY]: AWEykIdRr8h_fVHZ4rNu